src/test/software/amazon/event/ruler/ComparableNumberTest.java

```diff
@@ -139,4 +139,38 @@ public void epocTimestampRangeTest() throws Exception {
         }
         assertTrue(m.isEmpty());
     }
+
+    @Test()
+    @Ignore()
```

Comment:
Should this be removed?

Reply:
I was thinking we dont really need this test merge, my intention was only to validate the new code was performing better, but once we make the change I dont see the need to keep this benchmark at all

Reply:
Yup. I don't see value in this test at the moment. Happy to take it out.

Reply:
https://media2.giphy.com/media/6bJaVXMavwUTGZHFzf/giphy480p.mp4